game_master_node:
  ros__parameters:
    use_sim_time: True
    drone_detection_range: 137.0 # =height*55% for 2 horizontal corridors
    ship2ship_detection_range: 500.0 # Can see the other ship across the map
    ship2drone_detection_range: 162.0 # Can smaller objects only closer
    drone_communication_range: 144.0 # =drone_detection_range*105%
    ship_communication_range: 30.0 # =ship2ship_detection_range*105%
    drone_health: 1
    ship_health: 6
    drone_points: 10
    ship_points: 50
    game_duration: 200  # 5 minutes
    world_name: "swarmz_world_2"
    drone_max_speed: 12.0

warship_node:
  ros__parameters:
    use_sim_time: True
    canon_py_rotation_speed: 0.261 # pitch and yaw rotation 360°/24s -> 15°/s in rad/s


missile_server:
  ros__parameters:
    use_sim_time: True
    drone_missile_range: 69.0 # =drone_detection_range*50%
    ship_missile_range: 81.0 # =ship2drone_detection_range*50%
    drone_missile_damage: 1
    ship_missile_damage: 1
    drone_cooldown: 8.0
    ship_cooldown: 6.0
    drone_magazine: 2
    ship_magazine: 4
    laser_width: 3.0
    drone_padding_x: 0.5
    drone_padding_y: 0.5
    drone_padding_z: 0.5
    ship_padding_x: 6.0
    ship_padding_y: 1.0
    ship_padding_z: 1.0
    world_name: "swarmz_world_2"

kamikaze_server:
  ros__parameters:
    use_sim_time: True
    explosion_damage: 3
    explosion_range: 5.0
    world_name: "swarmz_world_2"